src/components/molecules/Form/Controls/Controls.components.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from 'react';
+import { BoxProps, Flex } from '../../../quarks';
+
+export const ControlsWrapper: FC<BoxProps> = props => (
+  <Flex sx={{ flexDirection: 'column', justifyContent: 'center' }} {...props} />
+);
+
+export const ControlsLabel: FC<BoxProps> = ({ checked, disabled, ...props }) => (
+  <Flex
+    as="label"
+    sx={{
+      alignItems: 'center',
+      mr: 2,
+      mb: 3,
+      ...(disabled && {
+        color: '2',
+        cursor: 'default',
+      }),
+      ...(!disabled && {
```

Comment:
tak samo jak wyżej, default niech będzie cursor: 'pointer', a gdy disabled to nadpisać, niepotrzebny warunek